packages/zowe-explorer/src/utils/SettingsConfig.ts

```diff
@@ -59,12 +57,8 @@ export class SettingsConfig {
         "Zowe-Jobs-Persistent": globals.SETTINGS_JOBS_HISTORY,
     };
     private static configurations = vscode.workspace.getConfiguration();
-    private static zoweOldConfigurations = Object.keys(SettingsConfig.configurations).filter((key) =>
-        key.match(new RegExp("Zowe-*|Zowe\\s*", "g"))
-    );
-    private static currentVersionNumber = semver.major(
-        vscode.extensions.getExtension("zowe.vscode-extension-for-zowe").packageJSON.version
-    );
+    private static zoweOldConfigurations = Object.keys(SettingsConfig.configurations).filter((key) => key.match(new RegExp("Zowe-*|Zowe\\s*", "g")));
+    private static currentVersionNumber = semver.major(vscode.extensions.getExtension("zowe.vscode-extension-for-zowe").packageJSON.version);
```

Comment:
Same question as above - should we split line 67 across multiple lines to make the linter happy?

Reply:
See above comment regarding lint warnings